pandas/tseries/tests/test_resample.py

```diff
@@ -1188,6 +1188,29 @@ def test_resample_loffset_count(self):
 
         assert_series_equal(result, expected)
 
+    def test_resample_loffset_arg_type(self):
+        # GH 13218, 15002
```

Comment:
would be better if you can put this in ``Base``, that way code is not duplicated. Of better to at least partially do it (e.g. if you have a specific part that *can* be put there, then add another test in the sub-classes to cover the rest).